### PR TITLE
Graciously handle timeout issues from CartoDB service

### PR DESCRIPTION
**Pull request by ikas**

### CODE REVIEW

File: app/src/routes/api/v1/prodesLoss.router.js [hunk 1]
```diff
@@ -13,9 +13,17 @@ class ProdesLossRouter {
...
-        const data = yield CartoDBService.getNational(this.params.iso, this.query.alertQuery, this.query.period);
-
-        this.body = ProdesLossSerializer.serialize(data);
+        try {
+            const data = yield CartoDBService.getNational(this.params.iso, this.query.alertQuery, this.query.period);
+            this.body = ProdesLossSerializer.serialize(data);
+        } catch (err) {
+            if (Array.isArray(err) && err.includes('You are over platform\'s limits: SQL query timeout error. Refactor your query before running again or contact CARTO support for more details.')) {
+                this.body = { errors: ['SQL query timeout error. Refactor your query and try running again.'] };
+                this.status = 504;
```
Comment (tiagojsag) [inline app/src/routes/api/v1/prodesLoss.router.js:22]: I think we should probably return the same http code as carto

File: app/src/routes/api/v1/prodesLoss.router.js [hunk 2]
```diff
@@ -13,9 +13,17 @@ class ProdesLossRouter {
...
+            } else {
+                this.body = {};
```
Comment (tiagojsag) [inline app/src/routes/api/v1/prodesLoss.router.js:24]: what happens here? I suspect this does not return a 4xx/5xx, but it also will not return the correct response

File: app/test/e2e/utils/mocks.js [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+                        gadm: '2.8',
+                        name: 'Brazil'
+                    }
+                }
+            }
+        });
+};
+
+const mockTimeoutCartoDBRequest = () => {
+    const base = config.get('cartoDB.apiUrl').replace('/api/v2/sql', '');
```
Comment (tiagojsag) [inline app/test/e2e/utils/mocks.js:423]: Couldn't we just build the url from the existing cartoDB.user config value?

File: app/src/routes/api/v1/prodesLoss.router.js (hunk 2 above)
Comment (ikas) [reply app/src/routes/api/v1/prodesLoss.router.js]: It was indeed returning 200, updated to return 500 Internal Server Error :+1: thanks for pointing out!

File: app/test/e2e/utils/mocks.js (hunk 3 above)
Comment (ikas) [reply app/test/e2e/utils/mocks.js]: Not 100% sure if that was possible, hence why I did it this way, but using CartoDB user also works :+1:

**Pull request merged**